src/Jackett.Common/Indexers/GreatPosterWall.cs

```diff
@@ -9,18 +9,18 @@
 using Jackett.Common.Utils;
 using Newtonsoft.Json.Linq;
 using NLog;
-
+using WebClient = Jackett.Common.Utils.Clients.WebClient;
```

Comment:
Why this over just `using Jackett.Common.Utils.Clients;`?

Reply:
`WebClient`  is also defined in  `System.Net`  package.

Reply:
I'm not a C# dev so I could use more details. Are you saying that you need to use the `using WebClient =` because otherwise there is a clash with WebClient  begin also defined in `system.net` ? Or have I the wrong end of the stick as usual?

Reply:
Yea that looks like the case - https://docs.microsoft.com/dotnet/api/system.net.webclient

We've used this workaround(?) in a few other places - https://github.com/Jackett/Jackett/search?q=System.Net+webclient
(Could I have found that out before asking the question? Maybe...)

Reply:
and I reply to myself: yes, without this you get `CS1503: Argument 6: cannot convert from 'System.Net.WebClient' to 'Jackett.Common.Utils.Clients.WebClient'`

Reply:
makes sense